ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -93,7 +93,7 @@ const AsyncVulnMgmtReports = asyncComponent(
 const AsyncVulnMgmtRiskAcceptancePage = asyncComponent(
     () => import('Containers/VulnMgmt/RiskAcceptance/RiskAcceptancePage')
 );
-const AsyncVulnMgmtPage = asyncComponent(() => import('Containers/Workflow/WorkflowLayout'));
+const AsyncVulnMgmtPage = asyncComponent(() => import('Containers/VulnMgmt/WorkflowLayout'));
```

Comment:
What was the value of moving all the Workflow folder files into the VulnMgmt folder?

Reply:
Astute question. After analysis that these files use only entity names and types for Vulnerability Management, which I have done at least one other time before this, moving them prevent cross-container imports now and the need to redo the analysis a third time in the future when we delete the container code. By similar reasoning, analysis of classic components will push down files imported by only one container. The recent deletion of non-postgres and Network Graph 1.0 code illustrate how hard it might be to schedule the work to finish a replacement.